Interoperability/python/qsharp/__init__.py

```diff
@@ -47,27 +49,24 @@
     SIMULATOR_COLOR = ca.Fore.BLUE + ca.Back.WHITE
     RESET_COLOR = ca.Style.RESET_ALL
 except:
-    ca = None
+    CA = None
```

Comment:
casing change ?

Reply:
In general, yes, but this is a constant providing a fallback for a missing package. PEP8 itself encourages breaking other PEP8 rules if there is a functional purpose served by doing so: https://www.python.org/dev/peps/pep-0008/#id15

Here, we can check `ca is None` before using colorama features.